Quantum-Control-Applications/Superconducting/Single Fixed Transmon/DRAG_calibration.py

```diff
@@ -0,0 +1,136 @@
+"""
+An experiment to calibrate the DRAG coefficient
```

Comment:
I would add here that you must have single shot readout right?

Reply:
Moreover, in this script you only calibrate drag_coef, or also drag_detun?
Is the sequence x180y90 - y180x90 come from a specific paper ? Why do we use this?

Reply:
This script only calibrates drag_coef, not drag_detun. For the latter it is another protocol.

This sequence was listed in Reed's thesis and a recent paper as well. I'm going to refer to them now.

single shot readout is not mandatory I will adjust with the readout macro

Reply:
Where is the readout macro defined ? (I did not see it in the program or the configuration)